README.md

```diff
@@ -159,6 +159,38 @@ Trigger a visualization for the track. This is typically album art.
 
 **This is an untested method.**
 
+### Queue
+`gmusic.queue` exposes methods to query and control the current queue
+
+#### `queue.getSongs()`
+Retrieve the songs currently in the queue
+
+**Returns:**
+
+- retVal `Array` - An array of song objects representing the songs currently in the queue.  Song objects have the following structure
+
+```
+{
+  id: String,         // A unique ID for this song
+  title: String,      // The songs title
+  album: String,      // The songs album
+  artist: String,     // The songs artist
+  art: String,        // A URL to the album art for this song
+  playing: Boolean,   // True if the song is currently playing, false otherwise
+  duration: Number    // Duration of song in milliseconds
+}
+```
+
+#### `queue.playSong(id)`
+Plays a song in the queue referenced via it's ID
+
+- id `String` - An ID string fetched through `queue.getSongs()`.  If the ID is no longer in the queue an error will be thrown
```

Comment:
We could use a comma in the second sentence here. Also, we should be using the same casing (i.e. `id`, not `ID`)

```
If the `id` is no longer in the queue, then an error will be thrown
```